### PR TITLE
Feder thesis numdata2

### PR DESCRIPTION
**Pull request by benjamin-feder**
numerical data lab update

### CODE REVIEW

File: 06_confidence_intervals/confidence_intervals.Rmd [hunk 1]
```diff
@@ -13,19 +13,20 @@ output:
...
 houses?". If you have access to only a sample of the population, as is often 
 the case, the task becomes more complicated. What is your best guess for the 
-typical size if you only know the sizes of several dozen houses? This sort of 
+typical size if you only know the sizes of several dozen houses? This type of 
 situation requires that you use your sample to make inference on what your 
 population looks like.
 
 <div id="boxedtext">
-**Setting a seed:** We will take some random samples and build sampling distributions
+**Setting a seed:** You will take some random samples and build sampling distributions
```
Comment (mine-cetinkaya-rundel) [inline 06_confidence_intervals/confidence_intervals.Rmd]: "some random samples" = "random samples"

File: 06_confidence_intervals/confidence_intervals.Rmd [hunk 2]
```diff
@@ -85,46 +95,52 @@ variable `area`.
...
-We can find the critical value for a 95% confidence interal using
-```{r z_star_95}
-z_star_95 <- qnorm(0.975)
-z_star_95
-```
-which is roughly equal to the value critical value 1.96 that you're likely
-familiar with by now.
+Although you can do this calculate by hand, the `infer` package provides a set of
+commands consistent with the `tidyverse` framework, thus making it easy to follow 
+a similar workflow. You can use these commands to find a single confidence
```
Comment (mine-cetinkaya-rundel) [inline 06_confidence_intervals/confidence_intervals.Rmd]: "similar workflow" = "similar syntax"

File: 06_confidence_intervals/confidence_intervals.Rmd [hunk 3]
```diff
@@ -85,46 +95,52 @@ variable `area`.
...
-z_star_95
-```
-which is roughly equal to the value critical value 1.96 that you're likely
-familiar with by now.
+Although you can do this calculate by hand, the `infer` package provides a set of
+commands consistent with the `tidyverse` framework, thus making it easy to follow 
+a similar workflow. You can use these commands to find a single confidence 
+interval:
+
+Command | Purpose
```
Comment (mine-cetinkaya-rundel) [inline 06_confidence_intervals/confidence_intervals.Rmd]: "Command" = "Function"

File: 06_confidence_intervals/confidence_intervals.Rmd [hunk 4]
```diff
@@ -85,46 +95,52 @@ variable `area`.
...
+------- | -------
+`specify` | Identify your variable of interest using the `formula` argument
+`generate` | The amount of repetitions you want to do
```
Comment (mine-cetinkaya-rundel) [inline 06_confidence_intervals/confidence_intervals.Rmd]: "amount of repetitions you want to do " = "number of samples you want to generate"

File: 06_confidence_intervals/confidence_intervals.Rmd [hunk 5]
```diff
@@ -85,46 +95,52 @@ variable `area`.
...
+`calculate` | The type of statistic you are looking for
```
Comment (mine-cetinkaya-rundel) [inline 06_confidence_intervals/confidence_intervals.Rmd]: "The type of statistic you are looking for" = "The sample statistic you want to do inference with, or you can also think of this as the population parameter you want to do inference for"

File: 06_confidence_intervals/confidence_intervals.Rmd [hunk 6]
```diff
@@ -152,30 +168,35 @@ params <- ames %>%
...
+Note: `qnorm` is set to .975 for a two-sided 95 percent confidence interval since
+the upper bound is at the 97.5 percentile.
 
 ```{r calculate-50-cis}
 ci <- ames %>%
         rep_sample_n(size = n, reps = 50, replace = TRUE) %>%
         summarise(x_bar = mean(area), 
                   se = sd(area) / sqrt(n),
-                  me = z_star_95 * se,
+                  me = qnorm(.975) * se,
```
Comment (mine-cetinkaya-rundel) [inline 06_confidence_intervals/confidence_intervals.Rmd]: Always use the leading 0s, e.g. 0.975

File: 06_confidence_intervals/confidence_intervals.Rmd [hunk 7]
```diff
@@ -199,17 +220,45 @@ The `ifelse` function is new. It takes three arguments: first is a logical state
...
-We now have all the information we need to create the plot. 
-Note that the `geom_errorbar()` function
-only understands `y` values, and thus we have used the `coord_flip()` function
-to flip the coordinates of the entire plot back to the more familiar vertical 
-orientation. 
+You now have all the information you need to create the plot, but you need to re-organize
+your data a bit for easy plotting. Specifically, you need to organize the data in a new
+data frame where each row represents one bound, as opposed to one interval. So this
+
+~~~
```
Comment (mine-cetinkaya-rundel) [inline 06_confidence_intervals/confidence_intervals.Rmd:226]: Why not show the real results here (by setting a seed first and `echo=FALSE`)?

File: 06_confidence_intervals/confidence_intervals.Rmd [hunk 8]
```diff
@@ -220,8 +269,8 @@ qplot(data = ci, x = replicate, y = x_bar, color = capture_mu) +
 
 ## More Practice
 
-1.  Pick a confidence level of your choosing, provided it is not 95%. What is 
-    the appropriate critical value?
+1.  Find a singular two-sided confidence interval with a percentage of your choosing
```
Comment (mine-cetinkaya-rundel) [inline 06_confidence_intervals/confidence_intervals.Rmd]: singular?

File: 07_inf_for_numerical_data/inf_for_numerical_data.Rmd [hunk 9]
```diff
@@ -13,20 +13,25 @@ output:
...
 ```
 
 ## Getting Started
 
 ### Load packages
 
 In this lab, we will explore and visualize the data using the `tidyverse` suite of 
-packages. The data can be found in the companion package for OpenIntro labs, `oilabs`.
+packages, and perform statistical inference using `infer`. The data can be found
+in the companion package for OpenIntro labs, `oilabs`.
```
Comment (mine-cetinkaya-rundel) [inline 07_inf_for_numerical_data/inf_for_numerical_data.Rmd]: Actually we're moving everything into the `openintro` package so you can update the language as such

File: 07_inf_for_numerical_data/inf_for_numerical_data.Rmd [hunk 10]
```diff
@@ -61,122 +67,143 @@ Remember that you can answer this question by viewing the data in the data viewe
...
+Using visualization and summary statistics, describe the distribution of weights.
+The `summary` function can be useful.
 
 ```{r summary}
-summary(nc$gained)
+summary(yrbss$weight)
 ```
 
-1.  How many mothers are we missing weight gain data from?
+1.  How many observations are we missing weights from?
```
Comment (mine-cetinkaya-rundel) [inline 07_inf_for_numerical_data/inf_for_numerical_data.Rmd:82]: Not totally opposed to this, but any chance one other variable might work, so we're not talking about weight?

File: 07_inf_for_numerical_data/inf_for_numerical_data.Rmd [hunk 11]
```diff
@@ -61,122 +67,143 @@ Remember that you can answer this question by viewing the data in the data viewe
...
-Next, we introduce a new function, `inference`, that we will use for conducting
-hypothesis tests and constructing confidence intervals. 
+After you have initialized the test, you need to simulate the test on the null
+distribution, which we will save as `null`.
 
-```{r inf-weight-habit-ht, tidy=FALSE}
-inference(y = weight, x = habit, data = nc, statistic = "mean", type = "ht", null = 0, 
-          alternative = "twosided", method = "theoretical")
+```{r inf-weight-habit-ht-null, tidy=FALSE, warning = FALSE}
+null <- yrbss %>%
```
Comment (mine-cetinkaya-rundel) [inline 07_inf_for_numerical_data/inf_for_numerical_data.Rmd]: Let's call this `null_dist`

File: 07_inf_for_numerical_data/inf_for_numerical_data.Rmd [hunk 12]
```diff
@@ -61,122 +67,143 @@ Remember that you can answer this question by viewing the data in the data viewe
...
 the one obtained in the previous exercise.
 
-8.  Conduct a hypothesis test evaluating whether the average weight gained by 
-younger mothers is different than the average weight gained by mature mothers.
+9.  Conduct a hypothesis test evaluating whether the average height is different
+for those who exercise at least three times a week and those who don't.
 
-9.  Now, a non-inference task: Determine the age cutoff for younger and mature 
-mothers. Use a method of your choice, and explain how your method works.
+10.  Now, a non-inference task: Determine the amount of different options there
```
Comment (mine-cetinkaya-rundel) [inline 07_inf_for_numerical_data/inf_for_numerical_data.Rmd]: amount = number

**Review by mine-cetinkaya-rundel — CHANGES_REQUESTED** [review]
I've left a few comments, throughout. I'll re-review when you push the changes.
